src/scripts/distribute_synapse_locs.py

```diff
@@ -29,6 +29,7 @@
 @click.option(
     "--mechanisms-path",
     "-m",
+    default="./mechanisms",
```

Comment:
@iraikov @frthjf Would it make sense to make this argument `required` instead of having default?

Reply:
I think `required` would mean that the user has to provide `-m` right? Up to you; I am not using the click interface at the minute.